### PR TITLE
Add the word "Variant" in bold before any games that aren't classical.

### PR DESCRIPTION
**Pull request by tttppp**
See #47: Hopefully this should warn new users when they aren't signing up for
standard Diplomacy.

### CODE REVIEW

**Comment by tttppp** [comment]
I tried to test this, but failed in two different ways.  First I tried simply installing this app and connecting to the production server. Google stops me from doing this - I get a log message "Login failed: Status{statusCode=DEVELOPER_ERROR, resolution=null}."

Secondly I tried to connect to a local server, but I've broken something locally and I'm not sure what.  I'm now getting the following error when trying to start diplicity locally:
```
dev_appserver.py --clear_datastore=yes --datastore_consistency_policy=consistent .
Traceback (most recent call last):
  File "/usr/lib/google-cloud-sdk/platform/google_appengine/dev_appserver.py", line 103, in <module>
    _run_file(__file__, globals())
  File "/usr/lib/google-cloud-sdk/platform/google_appengine/dev_appserver.py", line 97, in _run_file
    execfile(_PATHS.script_file(script_name), globals_)
  File "/usr/lib/google-cloud-sdk/platform/google_appengine/google/appengine/tools/devappserver2/devappserver2.py", line 322, in <module>
    main()
  File "/usr/lib/google-cloud-sdk/platform/google_appengine/google/appengine/tools/devappserver2/devappserver2.py", line 310, in main
    dev_server.start(options)
  File "/usr/lib/google-cloud-sdk/platform/google_appengine/google/appengine/tools/devappserver2/devappserver2.py", line 106, in start
    env_variables=parsed_env_variables)
  File "/usr/lib/google-cloud-sdk/platform/google_appengine/google/appengine/tools/devappserver2/application_configuration.py", line 851, in __init__
    env_variables)
  File "/usr/lib/google-cloud-sdk/platform/google_appengine/google/appengine/tools/devappserver2/application_configuration.py", line 131, in __init__
    self._config_path)
  File "/usr/lib/google-cloud-sdk/platform/google_appengine/google/appengine/tools/devappserver2/application_configuration.py", line 443, in _parse_configuration
    config.application = self._forced_app_id
  File "/usr/lib/google-cloud-sdk/platform/google_appengine/google/appengine/api/validation.py", line 361, in __setattr__
    value = self.GetValidator(key)(value, key)
  File "/usr/lib/google-cloud-sdk/platform/google_appengine/google/appengine/api/validation.py", line 599, in __call__
    return self.Validate(value, key)
  File "/usr/lib/google-cloud-sdk/platform/google_appengine/google/appengine/api/validation.py", line 881, in Validate
    return self.validator(value, key)
  File "/usr/lib/google-cloud-sdk/platform/google_appengine/google/appengine/api/validation.py", line 599, in __call__
    return self.Validate(value, key)
  File "/usr/lib/google-cloud-sdk/platform/google_appengine/google/appengine/api/validation.py", line 955, in Validate
    '\'%s\'' % (value, key, self.re.pattern))
google.appengine.api.validation.ValidationError: Value 'None' for application does not match expression '^(?:(?:[a-z\d\-]{1,100}\~)?(?:(?!\-)[a-z\d\-\.]{1,100}:)?(?!-)[a-z\d\-]{0,99}[a-z\d])$'
```

In conclusion please don't put this live until it can be tested :-)

**Comment by tttppp** [comment]
I think the second issue is because I updated the google cloud sdk to 157.0.0-0 (from 155.0.0-0).  I don't know why this would make a difference yet.

**Comment by tttppp** [comment]
I got my local server working by using `dev_appserver.py -A=diplicity .`  Previously I didn't need to specify a name of the app.

**Comment by tttppp** [comment]
I managed to get the app connected to the localhost server. When installing the modified version of the app then I hit the problem that I'm not allowed to log in to a Google account.  It seems like once that fails then nothing else works - even choosing "Local development mode" in the settings doesn't fix the issue.

I think perhaps I need to do something to disable the Google account sign in.

**Comment by zond** [comment]
When you have turned on local dev mode, select a "local dev mode fake id", it will make the server just accept that user ID, and not even try to log in via Google.

**Comment by tttppp** [comment]
Yep, I did that, but before you can get to the settings you have to select a Google account. This seems to cause the app to get into a loop of failed logins.

I'll have a go at modifying the start up code locally when I get a chance.

**Comment by zond** [comment]
Ah, but log in once to the regular server before you do this?

**Comment by tttppp** [comment]
It doesn't seem to work either way around. I can't authenticate with Google, and consequently can't save changes to my preferences. When I set the local dev mode fake id then it doesn't affect the URL for connecting to the game server.

**Comment by zond** [comment]
Hm, right. 

To Google authenticate with your local server maybe you need to do step 7 from the diplicity README.md (configure the server). The relevant part would be to create a Google Cloud project, and then use the client ID and secret from it - that should make it possible to authenticate with Google OAuth.

But - with fake ID set up I really don't think that should be necessary :O

Note that you can sort of break out of the login cycle (which sort of breaks the regular flow, which is problematic and may need app restart) by clicking outside the login dialogs.

Then you can go to settings and change to fake ID, then kill/restart the app. Try that! :)

**Comment by zond** [comment]
Also, getting it to work with prod server should be simple :O

Don't understand why that doesn't work - it's the most straightforward way to test that kind of pure client change..

**Comment by tttppp** [comment]
Ok! I got this to work by hacking the DEFAULT_URL to be localhost and changing a few other bits of code.  It works :-)

The question now is whether this is an obvious enough hint for new users:

variantlabel

I'm happy to try some other styling if you have ideas.

**Comment by zond** [comment]
Sounds good to me, and if people are unhappy it can be changed later :)

**Comment by tttppp** [comment]
Great - thanks!
